api/enable_column_stats.md

```diff
@@ -0,0 +1,80 @@
+---
+api_name: enable_column_stats()
+excerpt: Enable range tracking for columns of chunks from a hypertable  
+topics: [hypertables]
+keywords: [hypertables, chunks, range-tracking]
+tags: [columns, ranges, min-max, chunks]
+api:
+  license: apache
+  type: function
+---
+
+# enable_column_stats()
+
+Allow users to specify that ranges (min/max values) be tracked
+for a specific column of chunks belonging to a hypertable.
+
+### Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`hypertable`|REGCLASS|Hypertable to which the column belongs to|
+|`column_name`|TEXT|Column to track range statistics for|
+
+### Optional arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`if_not_exists`|BOOLEAN|Set to true to avoid throwing an error if ranges are already being tracked for the column. A notice is issued instead. Defaults to false|
```

Comment:
```suggestion
|`if_not_exists`|BOOLEAN|Set to `true` so that a notice is sent when ranges are not being tracked for a column. By default, an error is thrown.| |
```

Reply:
@billy-the-fish done.